circuit-codegen/src/main/kotlin/com/slack/circuit/codegen/CircuitSymbolProcessorProvider.kt

```diff
@@ -55,9 +47,17 @@ import dagger.assisted.AssistedFactory
 import javax.inject.Inject
 import javax.inject.Provider
 
-private val CIRCUIT_INJECT_ANNOTATION = CircuitInject::class.java.canonicalName
-private val CIRCUIT_PRESENTER = Presenter::class.java.canonicalName
-private val CIRCUIT_UI = Ui::class.java.canonicalName
+private val MODIFIER = ClassName("androidx.compose.ui", "Modifier")
+private val CIRCUIT_INJECT_ANNOTATION =
+  ClassName("com.slack.circuit.codegen.annotations", "CircuitInject")
+private val CIRCUIT_PRESENTER = ClassName("com.slack.circuit", "Presenter")
```

Comment:
nit: we're using `com.slack.circuit` a number of times here and in a couple `MemberName`'s below. Consider making it a constant to avoid any typos now and in the future

Reply:
[a326a27](https://github.com/slackhq/circuit/pull/422/commits/a326a27f74c62e160265779d07cd3f29257f10f8)